requirements.txt

```diff
@@ -1,30 +1,2 @@
-Django>=1.9,<1.10
-cookiecutter==1.0.0
-psycopg2cffi
-raven==5.0.0
-markdown
-django-libsass
-certifi
-dj-database-url
-django-cas-ng==3.5.4
-django-mptt==0.8.5
-requests
-elastic-git
-unicore.content
-wagtail==1.9
-responses
-babel
-wagtailmedia==0.1.3
-django-extensions
-celery<4.0
-redis
-six>=1.9
-django-google-analytics-app==2.1.4
-mote-prk==0.2.1
-beautifulsoup4
-ujson==1.35
-djangorestframework==3.3.2
-cached-property
-setuptools==18.5
-molo.profiles<6.0.0,>=5.0.0
-django-el-pagination==3.1.0
+# Our dependencies are all specified in setup.py
```

Comment:
For posterity/transparency, here is an explanation:
```
Jamie Hewland [10:21 AM] 
jerith: random question you have probably been asked at least a few times before: what are your primary reasons for keeping dependencies in `setup.py` rather than `requirements.txt`?.. Mine is that it’s a bit of a hack to read the lines from requirements.txt and interpret them as a list of requirements as they are actually a list of instructions to `pip` which could involve things that aren’t simple dependencies.

Jeremy Thurgood [10:23 AM] 
jamie: Firstly, `setup.py` is the place the installer gets the deps from, so it's always the first place I look when I want to find out what deps a thing has.

[10:24] 
Secondly, requirements files and `setup.py` have different syntax, and it's really hard to make sure that whatever reads the requirements file gets everything right.

[10:25] 
Thirdly, the requirements file is a separate thing that needs to be properly packaged in an sdist or something, and that's not an obvious step. (Especially when everything works locally, because you're installing from your repo clone rather than the sdist or whatever.)

[10:26] 
There are a few more, but those are the ones I can think of offhand.

[10:26] 
Or rather, those are the practical reasons.

[10:27] 
The more philosophical reason is that `setup.py` tells you what your dependencies are, and should thus be as broad as possible. `requirements.txt` is more about specific deployments or whatever (which is why we have `requirements-dev.txt`, `requirements-doc.txt`, etc.) and can have extra things or more tightly constrained versions.

[10:28] 
For example, a Django app might have `Django` in its `setup.py`, but `gunicorn` in its `requirements.txt`.
```